### PR TITLE
Filled in placeholders for name and project folder

### PR DESCRIPTION
**Pull request by elinwub**
I messed up a little bit trying to follow the lecture on friday, but after watching the videos I hope it turned out right

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -4,7 +4,7 @@ version = 0.0.1
 description = Exersices for INF200
 author = Elin Bjoernson
 author_email = elinbj@nmbu.no
-
+
```
Comment (huynhngoc) [inline setup.cfg]: Please remove any trailing white-space that makes files without actual changes are added in the pull requests.

**Review by huynhngoc — CHANGES_REQUESTED** [review]
Nice tried. Although messed up at the beginning but you finished it. Remember not to make the same mistake again:
- Don't split up pull requests for one exercise (especially one has been closed before it is approved) as the reviewer can get confused
- Watch out trailing spaces

You just need to fix the trailing space then I can approve the PR.
Note that in the next exercises if you make the same mistake then you can fail those exercises.

File: setup.cfg (hunk 1 above)
Comment (elinwub) [reply setup.cfg]: I think I have removed trailing white space now

**Review by huynhngoc — APPROVED** [review]
Nice!

**Pull request merged**